src/main/java/com/derongan/minecraft/mineinabyss/MineInAbyss.java

```diff
@@ -49,31 +59,39 @@ public void onEnable() {
 
         //Vault setup
         if (!setupEconomy()) {
-            getLogger().severe(String.format("[%s] - Disabled due to no Vault dependency found!", getDescription().getName()));
+            getLogger().severe(String
+                    .format("[%s] - Disabled due to no Vault dependency found!", getDescription().getName()));
             getServer().getPluginManager().disablePlugin(this);
             return;
         }
 
         //Geary setup
         if (setupGeary()) {
-            ItemStack itemStack = new ItemStack(Material.DIAMOND_SHOVEL);
-            ItemMeta itemMeta = itemStack.getItemMeta();
-            itemMeta.setCustomModelData(3);
-            itemMeta.setDisplayName("Grappling Hook");
-            itemStack.setItemMeta(itemMeta);
-            ShapedRecipe recipe = gearyService.createRecipe(new NamespacedKey(this, "grappling_hook"),
-                () -> PredefinedArtifacts
-                    .createGrapplingHook(1.3, 3, 4, Color.fromRGB(142, 89, 60), 1, 64), itemStack);
-
-            recipe.shape("III", "ISI", " S ");
-            recipe.setIngredient('I', Material.IRON_INGOT);
-            recipe.setIngredient('S', Material.STRING);
-
-            getServer().addRecipe(recipe);
+            NamespacedKey grapplingRecipeKey = new NamespacedKey(this, "grappling_hook");
+            NamespacedKey starCompassRecipeKey = new NamespacedKey(this, "star_compas");
```

Comment:
this should totally be a json file

Reply:
Totally should be- I have a bunch of code from before geary was nbt based that allowed defining prefabs/recipes in geary. I'm repurposing it soon tm